crates/ruff/src/rules/pycodestyle/helpers.rs

```diff
@@ -25,6 +25,7 @@ pub(super) fn is_overlong(
     limit: usize,
     ignore_overlong_task_comments: bool,
     task_tags: &[String],
+    tab_size: usize,
```

Comment:
Nit: Should we introduce a `TabSize`newtype wrapper (using `u32` or even a `u16` or `u8` should probably be sufficient) to pass a more meaningful type?